### PR TITLE
structurizr-cli 1.4.4 (new formula)

### PR DESCRIPTION
**Pull request by gilbertotcc**
Add Structurizr CLI (https://github.com/structurizr/cli), the
command line utility for Structurizr.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/structurizr-cli.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    libexec.install "structurizr-cli-#{version}.jar" => jar
+    (bin/"structurizr-cli").write <<~EOS
+      #!/bin/bash
+      exec "#{Formula["openjdk"].opt_bin}/java" -jar #{libexec}/#{jar} "$@"
+    EOS
+    chmod 0755, bin/"structurizr-cli"
+  end
+
+  test do
+    shell_output bin/"structurizr-cli", 1
```
Comment (SMillerDev) [inline Formula/structurizr-cli.rb:23]: Please use `assert_match` to check for some actual output here

File: Formula/structurizr-cli.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    jar = "structurizr-cli.jar"
+    libexec.install "structurizr-cli-#{version}.jar" => jar
+    (bin/"structurizr-cli").write <<~EOS
```
Comment (SMillerDev) [inline Formula/structurizr-cli.rb:15]: We have a convenience function for this (I believe it's `write_jar_wrapper`) please use that.

Comment (gilbertotcc) [reply Formula/structurizr-cli.rb]: Not really `write_jar_wrapper` but I've found the solution looking in *similar* formulae. Thanks for the suggestion.

File: Formula/structurizr-cli.rb (hunk 1 above)
Comment (gilbertotcc) [reply Formula/structurizr-cli.rb]: I've used `assert_equal` now, please, check if the test looks good now. I took inspiration from an existing formula.

File: Formula/structurizr-cli.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  def install
+    jar = "structurizr-cli-#{version}.jar"
+    libexec.install jar
+    bin.write_jar_script libexec/jar, "structurizr-cli"
+  end
+
+  test do
+    expected_output = <<~EOS.strip
+      Structurizr CLI v1.4.4
```
Comment (SMillerDev) [inline Formula/structurizr-cli.rb:20]:
```suggestion
      Structurizr CLI v#{version}
```

File: Formula/structurizr-cli.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    jar = "structurizr-cli-#{version}.jar"
+    libexec.install jar
+    bin.write_jar_script libexec/jar, "structurizr-cli"
```
Comment (SMillerDev) [inline Formula/structurizr-cli.rb:15]:
```suggestion
    libexec.install "structurizr-cli-#{version}.jar"
    bin.write_jar_script libexec"/structurizr-cli-#{version}.jar", "structurizr-cli"
```

**Comment by gilbertotcc** [comment]
@SMillerDev Thanks for the suggestions. They were very helpful. Sorry for my not perfect PR, but I'm new in writing Homebrew formulae.

I've applied your changes. Hoping now the PR is fine.

**Comment by SMillerDev** [comment]
> @SMillerDev Thanks for the suggestions. They were very helpful. Sorry for my not perfect PR, but I'm new in writing Homebrew formulae.

No problem, we all have to start somewhere.

**Comment by SMillerDev** [comment]
Looks good, can you squash your commits into one matching the pull request name

**Comment by gilbertotcc** [comment]
@SMillerDev Squashed! Waiting for checks.

Thanks for your patience again.

**Review by SMillerDev — APPROVED** [review]
Thanks @gilbertotcc  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**